ouroboros-consensus-cardano/test/Test/ThreadNet/Cardano.hs

```diff
@@ -770,8 +771,14 @@ mkProtocolCardanoAndHardForkTxs
   Constants
 -------------------------------------------------------------------------------}
 
+-- | The active slot coefficient, @f@.
+--
+-- Some of these tests includes Shelley epochs in which stakepools are actually
+-- leading. In that case, the @k@, @d@, and @f@ parameters and the length of
+-- any scheduled network partitions need to be balanced so that Common Prefix
+-- violations (in particular, wedges) are extremely unlikely.
 activeSlotCoeff :: Rational
-activeSlotCoeff = 0.5
+activeSlotCoeff = 0.2   -- chosen to match mainnet
```

Comment:
I'm not sure I understand "chosen to match mainnet", because mainnet uses 0.05. Or is that "relative"?

Reply:
Yeah, typo on my part. Thanks for catching.